queries/ne.jinja2

```diff
@@ -268,7 +268,7 @@ FROM
 WHERE
 
   {{ bounds['line']|bbox_filter('the_geom', 3857) }}
-  AND scalerank <= {{ zoom }}
+  AND {{ zoom }} >= 5 AND scalerank <= {{ zoom }}
```

Comment:
Good point! Fixed in 5208033.

I moved the condition to the `{% if ... %}` static conditional block, as experimentally it seems as if `SELECT * FROM table WHERE FALSE` still executes the scan on `table`, which I had assumed it wouldn't.